sdk/formrecognizer/ai-form-recognizer/src/formTrainingClient.ts

```diff
@@ -678,7 +778,37 @@ export class FormTrainingClient {
 }
 
 /**
- * @private
+ * @internal
+ */
+async function composeModelInternal(
+  // eslint-disable-next-line @azure/azure-sdk/ts-use-interface-parameters
+  client: GeneratedClient,
```

Comment:
I guess you do not have time now to refactor the LRO logic here similar to what was done in TA and KV but perhaps you can open an issue for it?

Reply:
I did do some refactoring along those lines for the recognition pollers. I would like to consider doing the same for these pollers as well, but I did remove `client` from the poller state. The `client` you see here is bound to the poller lexically and so won't be exposed through the state.